src/components/CarouselImage/pageObject/index.js

```diff
@@ -21,6 +21,17 @@ class PageCarouselImage {
             .$('a')
             .click();
     }
+
+    /**
+     * Returns the header of the CarouselImage.
+     * @method
+     * @returns {string}
+     */
+    getHeader() {
+        return $(this.rootElement)
+            .$('[title="Imagen Header"]')
+            .getHTML(false);
```

Comment:
here you can use getText

Reply:
The problem with getText () is that if the element is not visible the function returns empty.